app/web/src/components/AttributesPanel/AttributesPanelItem.vue

```diff
@@ -861,6 +860,8 @@ const sourceTooltip = computed(() => {
       return `${propName.value} is set by a dynamic function`;
     } else if (propHasSocket.value) {
       return `${propName.value} is set via an empty socket`;
+    } else if (propControlledByParent.value) {
+      return `${propName.value} is set via a function from an ancestor`;
```

Comment:
The `PropControlledByParent` check needs to execute before the `propSetByDynamicFunc`, since everything that is controlled by parent is set by dynamic func but the opposite is not true

Reply:
Just noticed this fact means you can simplify `propIsEditable`

Reply:
perfect, thank you!!